### PR TITLE
[MINOR] Doc fix for spark interpreter

### PR DESCRIPTION
**Pull request by zjffdu**
### What is this PR for?
Several changes on doc of spark interpreter. 

* %spark, %sql, %pyspark only works when spark is the default interpreter group of note. So I update the doc to use the full interpreter name.
* Add SparkSession for 2.0
* Also add comments inline with other changes to explain the reason. 

### What type of PR is it?
[Documentation]

### Todos
* [ ] - Task

### What is the Jira issue?
* No jira created. 

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: docs/interpreter/spark.md [hunk 1]
```diff
@@ -190,29 +191,24 @@ Once `SPARK_HOME` is set in `conf/zeppelin-env.sh`, Zeppelin uses `spark-submit`
   <tr>
     <th>spark-defaults.conf</th>
     <th>SPARK_SUBMIT_OPTIONS</th>
-    <th>Applicable Interpreter</th>
```
Comment (zjffdu) [inline docs/interpreter/spark.md]: AFAIK, there settings are applicable for all the spark interpreters. So I remove this column.

File: docs/interpreter/spark.md [hunk 2]
```diff
@@ -190,29 +191,24 @@ Once `SPARK_HOME` is set in `conf/zeppelin-env.sh`, Zeppelin uses `spark-submit`
...
     <td>Comma-separated list of maven coordinates of jars to include on the driver and executor classpaths. Will search the local maven repo, then maven central and any additional remote repositories given by --repositories. The format for the coordinates should be groupId:artifactId:version.</td>
   </tr>
   <tr>
     <td>spark.files</td>
     <td>--files</td>
-    <td>%pyspark</td>
     <td>Comma-separated list of files to be placed in the working directory of each executor.</td>
   </tr>
 </table>
-> Note that adding jar to pyspark is only availabe via `%dep` interpreter at the moment.
```
Comment (zjffdu) [inline docs/interpreter/spark.md]: I think these setting are also applicable for pyspark, so remove this line.

**Comment by zjffdu** [comment]
\cc @Leemoonsoo @AhyoungRyu

**Comment by AhyoungRyu** [comment]
@zjffdu Thank you for updating Spark interpreter docs. Looks great I think 👍 
While I look through current Spark documentation page to review this PR, I found some typos, grammar errors and removed useless markdown escaping. So I fixed them based on your branch. If you don't mind, could you check [https://github.com/zjffdu/zeppelin/pull/1](https://github.com/zjffdu/zeppelin/pull/1) out?

**Comment by AhyoungRyu** [comment]
@zjffdu Did you close this PR on purpose?

**Comment by zjffdu** [comment]
Sorry @AhyoungRyu  I didn't notice you PR is based on my branch and thought you will merge to master directly.

**Comment by AhyoungRyu** [comment]
@zjffdu It's okay :) LGTM 👍

**Comment by AhyoungRyu** [comment]
Merging into master if there are no more discussions

**Pull request closed without merging**